src/structstore.hpp

```diff
@@ -214,6 +214,10 @@ class StructStore {
     }
 };
 
+bool operator== (const StructStore& s1, const StructStore& s2) {
```

Comment:
```suggestion
bool operator==(const StructStore& s1, const StructStore& s2) {
```

Reply:
Maybe remove the comparison operator, see comments below

Reply:
Ok, removed the operator.